BUILD.md

```diff
@@ -98,20 +107,13 @@ Link-time optimizations provide a performance improvement. You should strongly c
 
 - If you are on MacOS, turning on LTO means that if you upgrade your version of XCode, you will have to rebuild your Pony compiler. You won't be able to link Pony programs if there is a mismatch between the version of XCode used to build the Pony runtime and the version of XCode you currently have installed.
 
-You can enable LTO when building the compiler in release mode. There are slight differences between platforms so you'll need to do a manual setup. LTO is enabled by setting `lto` to `yes` in the build command line like:
+LTO is enabled by setting `lto` to `yes` in the build command line like:
 
 ```bash
-make lto=yes
+make configure lto=yes
+make build
 ```
 
-If the build fails, you have to specify the LTO plugin for your compiler in the `LTO_PLUGIN` variable. For example:
-
-```bash
-make LTO_PLUGIN=/usr/lib/LLVMgold.so
-```
-
-Refer to your compiler documentation for the plugin to use in your case.
-
 ## runtime-bitcode
 
 If you're compiling with Clang, you can build the Pony runtime as an LLVM bitcode file by setting `runtime-bitcode` to `yes` in the build command line:
```

Comment:
When I try to run `make runtime-bitcode=yes` or `make configure runtime-bitcode=yes`, I get:
```
Makefile:76: *** recipe commences before first target.  Stop.
```

Reply:
OK, I have fixed the tabs-vs-spaces in the Makefile.

Reply:
Strange, when I run that on either Ubuntu or Fedora it works for me.  I'll poke around a bit more.

Reply:
Here's the two log files for that run: https://gist.github.com/jemc/97907246186b22e18abee10559603893

Reply:
Were you using gcc before and then switched to clang?  You might need to do a `make clean` in between?  Because the CMake log is trying to use gcc throughout.

Reply:
CMake caches a lot of configuration.

Reply:
Thanks, `make config=debug clean` did it for me. I didn't realize that earlier `make configure` calls would affect later ones.